common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -3250,7 +3251,7 @@ def _new_block(self, user_id, category, block_fields, definition_id, new_id, raw
         if not raw:
             block_fields = self._serialize_fields(category, block_fields)
         if not asides:
-            asides = []
+            asides = {}
```

Comment:
I know this was missing from the docs already, but could you add `asides` to the docstring, so that it's clearer what kind of data is being passed?